pkgs/top-level/all-packages.nix

```diff
@@ -10273,13 +10273,9 @@ in
               then callPackage ../development/libraries/gnu-efi { }
             else null;
 
-  gnutls = callPackage
-    (if stdenv.isDarwin
-      # Avoid > 3.5.10 due to frameworks for now; see discussion on:
-      # https://github.com/NixOS/nixpkgs/commit/d6454e6a1
-      then ../development/libraries/gnutls/3.5.10.nix
```

Comment:
You will need to remove this file now as it is not referenced by anything else.

Reply:
good catch. Fixed, thanks!